src/services/options_interface.ts

```diff
@@ -29,6 +29,9 @@ export interface OptionDefinitions extends KeyboardShortcutsOptions<KeyboardActi
     detailFontFamily: FontFamily;
     monospaceFontFamily: FontFamily;
     spellCheckLanguageCode: string;
+    // Share settings
```

Comment:
can't we use boolean here?
would be more fitting for a "on/off" type checkbox, like it is 

or is there a special reason to use string here, that I might be missing?

Reply:
oof, that was just me not thinking clearly